models/examples/02_dto_examples.py

```diff
@@ -0,0 +1,108 @@
+from credmark.cmf.model import Model
+from credmark.cmf.types import Portfolio
+from credmark.dto import DTO, DTOField, EmptyInput
+from models.dtos.example import ExampleModelOutput
+
+
+@Model.describe(slug='example.dto',
+                version='1.0',
+                display_name='Test Model',
+                description='Framework Test Model',
+                developer='Credmark',
+                input=EmptyInput,
+                output=ExampleModelOutput)
+class DtoExampleModel(Model):
+    def run(self, _) -> ExampleModelOutput:
+        output = ExampleModelOutput(
+            github_url="https://github.com/credmark/credmark-models-py/blob/main/models/examples/02_dto_examples.py",
+            documentation_url="https://developer-docs.credmark.com/en/latest/components.html#data-transfer-object-dto")
+
+        output.log("DTOs are classes with typed properties which will serialize and deserialize to and from JSON.")
+
+        class Token(DTO):
+            symbol: str
+            total_supply: int = 1000
+
+        output.log("""
+class Token(DTO):
+    symbol: str
+    total_supply: int = 1000""")
```

Comment:
I feel like putting in a fake total_supply will be confusing

Reply:
Wanted to a demonstrate a field with default value. I'll replace it with `decimals: int = 18`